ticket4uApp/t4uApp/models/concert/model.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+from django.utils import timezone
+from t4uApp.utils import upload_to
+from django.db.models.signals import post_delete
+from django.dispatch.dispatcher import receiver
+from django.core.validators import FileExtensionValidator
+
+
+class ConcertType(models.Model):
+    title = models.CharField(max_length=100)
+
+
+class Place(models.Model):
+    address = models.CharField(max_length=100, default='')
+    latitude = models.CharField(max_length=100)
+    longitude = models.CharField(max_length=100)
+
+
+class Concerts(models.Model):
+    title = models.CharField(max_length=100)
+    date = models.DateTimeField(default=timezone.now)
+    place = models.ForeignKey(Place, null=True, on_delete=models.SET_NULL)
+    type = models.ForeignKey(ConcertType, null=True, on_delete=models.SET_NULL)
+    poster = models.FileField(
+        upload_to=upload_to,
+        validators=[FileExtensionValidator(allowed_extensions=["png", "jpg", "jpeg"])],)
+    desc = models.CharField(max_length=15000, null=True, blank=True)
+    price = models.IntegerField(default=0)
+    ticket = models.IntegerField(default=0)
+
+    def __str__(self):
+        return self.title
+
+
+class ConcertParty(Concerts):
```

Comment:
bad naming `ConcertParty`, we don't say `CarAudi` we just leave Audi... the same for another concert types